src/main/java/ch/uzh/ifi/hase/soprafs24/service/GameService.java

```diff
@@ -92,6 +93,17 @@ public void startGame(Integer gameId) {
         }
     }
 
+    private Integer setNewGameStatsId(){
+        // search for Stats & inMemoryGameRepository => find largest gameStatsID.
+        Integer gameStatsId = Integer.max(inMemoryGameRepository.getLatestGameStatsId(), statsService.getLatestGameId());
+        gameStatsId++;
+
+        // check gameStatsId is unique value
+        assert !statsRepository.existsByGameId(gameStatsId);
+        assert !inMemoryGameRepository.existsByGameStatsId(gameStatsId);
+        return gameStatsId;
```

Comment:
I think we shouldn't have these assertions without catching and handling them if they occur, as the user cannot influence this, right?

Reply:
yes, it is unnecessary. I will delete this and pull